clippy_lints/Cargo.toml

```diff
@@ -28,6 +28,7 @@ smallvec = { version = "1", features = ["union"] }
 toml = "0.5.3"
 unicode-normalization = "0.1"
 semver = "0.11"
```

Comment:
can we remove the dependency on `semver` now?

Reply:
I don't think so. It is still used in the ``wildcard_dependencies`` cargo lint